setupbase.py

```diff
@@ -296,14 +296,14 @@ def run(self):
             self.__print('New hash is: %s' % md5)
 
         # Now it's time to patch "iconic_font.py":
```

Comment:
Should be `__init__.py`

Reply:
Fixed at 0ae5d6bcf298f30747ad6086f19f25dfa08161c9